token-bridge/README.md

```diff
@@ -282,13 +279,13 @@ When running the processes, the following commands can be used to test functiona
 
 ## Contributing
 
-See the [CONTRIBUTING](CONTRIBUTING.md) document for contribution, testing and pull request protocol.
+See the [CONTRIBUTING](../CONTRIBUTING.md) document for contribution, testing and pull request protocol.
 
 ## License
 
 [![License: LGPL v3.0](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
 
-This project is licensed under the GNU Lesser General Public License v3.0. See the [LICENSE](LICENSE) file for details.
+This project is licensed under the GNU Lesser General Public License v3.0. See the [LICENSE](../LICENSE) file for details.
 
 ## References
```

Comment:
Let's move this section to the main README?

Reply:
Done.